wazo_calld/plugins/voicemails/services.py

```diff
@@ -81,17 +84,40 @@ def get_greeting(self, voicemail_id, greeting):
                 raise NoSuchVoicemailGreeting(greeting)
             raise
 
+    def create_greeting(self, voicemail_id, greeting, data):
+        vm_conf = confd.get_voicemail(voicemail_id, self._confd_client)
+        body = {
+            'greeting_base64': base64.b64encode(data).decode()
+        }
+        try:
+            self._ari.wazo.createVoicemailGreeting(
+                context=vm_conf['context'],
+                voicemail=vm_conf['name'],
+                greeting=greeting,
+                body=body
+            )
+        except ARIHTTPError as e:
+            # FIXME(sileht): Should be 409 or 400
```

Comment:
Why is not the case? not merged or it's a bug in the asterisk patch?

Reply:
done